src/usdb_syncer/song_txt/__init__.py

```diff
@@ -49,6 +49,19 @@ def parse(cls, value: str, logger: Log) -> SongTxt:
         lines = [line for line in value.splitlines() if line]
         headers = Headers.parse(lines, logger)
         meta_tags = MetaTags.parse(headers.video or "", logger)
+        if headers.videogap is not None:
```

Comment:
This would be better placed in some sanitation code than in the parse code.

Reply:
Ah, good point! I moved it to a `fix_videogap()` sanitation function.